tasks/main.yml

```diff
@@ -0,0 +1,33 @@
+---
+- name: Install required packages
+  apt:
+    name: "{{ item }}"
+  with_items:
+    - python
+    - python-flask
+    - python-requests
+    - python-pillow
+
+- name: Clone repository
+  git:
+    repo: 'https://github.com/softdevteam/mattermost-github-integration.git'
+    dest: "{{ mattermost_github_integration_install_path }}"
+    depth: 1
```

Comment:
Why don't you add `update: true`? And what if I want to fix the version? Please supply a revision attribute that defaults to master

Reply:
fixed

Reply:
Why do you always lie to me? :/